packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -1368,6 +1372,11 @@ export const CURVE_MAINNET_INFOS: { [name: string]: CurveInfo } = {
         pool: CURVE_POOLS.stgusdc,
         gasSchedule: 250e3,
     }),
+    [CURVE_POOLS.cvxfxs]: createCurveFactoryCryptoExchangePool({
+        tokens: [MAINNET_TOKENS.FXS, MAINNET_TOKENS.cvxFXS],
+        pool: CURVE_POOLS.cvxfxs,
+        gasSchedule: 250e3,
```

Comment:
We don't always completely nail this (since it's hard to automate) but it kind of looks like this costs a bit more (`390e3`) than typical (example [tx](https://etherscan.io/tx/0x144041f5baa2be3bfa92daad5d15973385cb91fa2d5dac81d8b1db2dbb426228)). FXS looks like a bit of a whopper with a transfer cost of 80k.

<img width="449" alt="image" src="https://user-images.githubusercontent.com/27389/165646984-36977de3-9b63-468a-9a1a-672c6ab38ce4.png">

My [preferred tool is Ethtx](https://ethtx.info/mainnet/0x144041f5baa2be3bfa92daad5d15973385cb91fa2d5dac81d8b1db2dbb426228/) to get a readable trace.

Reply:
Oh, that's interesting! Thanks for the link -- it looks really handy!